bundles/AdminBundle/DependencyInjection/Configuration.php

```diff
@@ -54,6 +54,10 @@ public function getConfigTreeBuilder(): TreeBuilder
                     ->end()
                 ->end()
             ->end()
+            ->arrayNode('admin_csp_header')
+                ->canBeEnabled()
+                ->info('Can be used to enable or disable the Content Security Policy headers.')
+            ->end()
```

Comment:
I think it would be cool here, if you could configure additional URLs.
That would make it very easy for bundles to add their custom stuff (without using a custom compiler pass). 
Example: 
```yaml
pimcore_admin:
    admin_csp_header:
        additional_urls:
            script_src: 
                - "https://oreo.cat/scripts/meow.js"
                - "https://bagheera.cat/*"
```

Reply:
Added configuration node to define additional urls and am internal compiler pass which adds it.